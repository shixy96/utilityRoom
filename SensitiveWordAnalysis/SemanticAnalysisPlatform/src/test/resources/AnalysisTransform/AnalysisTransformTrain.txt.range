x
-1 1
1 0 74
2 0 36
3 0 54
4 0 93
5 0 93
6 0 162
7 0 246.60000000000002
8 0 216
9 0 364.60000000000002
10 0 486
11 0 498
12 0 159.19999999999999
13 0 207
14 0 219
15 0 165
16 0 134.40000000000001
17 0 170
18 0 225
19 0 195
20 0 3
21 0 24
