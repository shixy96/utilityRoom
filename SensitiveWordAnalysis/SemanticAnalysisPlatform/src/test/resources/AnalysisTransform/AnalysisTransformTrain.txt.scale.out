log2c=5 log2g=-7 rate=76.676
log2c=-1 log2g=-7 rate=74.5969
log2c=5 log2g=-1 rate=81.7367
log2c=-1 log2g=-1 rate=78.4936
log2c=11 log2g=-7 rate=78.8671
log2c=11 log2g=-1 rate=83.9776
log2c=5 log2g=-13 rate=74.5969
log2c=-1 log2g=-13 rate=74.5969
log2c=11 log2g=-13 rate=76.5142
log2c=-3 log2g=-7 rate=74.5969
log2c=-3 log2g=-1 rate=76.1594
log2c=-3 log2g=-13 rate=74.5969
log2c=5 log2g=1 rate=83.9029
log2c=-1 log2g=1 rate=80.4793
log2c=11 log2g=1 rate=86.1936
log2c=-3 log2g=1 rate=79.1783
log2c=9 log2g=-7 rate=78.3131
log2c=9 log2g=-1 rate=83.3862
log2c=9 log2g=-13 rate=74.2795
log2c=9 log2g=1 rate=85.6147
log2c=5 log2g=-11 rate=74.6218
log2c=-1 log2g=-11 rate=74.5969
log2c=11 log2g=-11 rate=77.479
log2c=-3 log2g=-11 rate=74.5969
log2c=9 log2g=-11 rate=76.5142
log2c=3 log2g=-7 rate=74.1923
log2c=3 log2g=-1 rate=80.0622
log2c=3 log2g=-13 rate=74.5969
log2c=3 log2g=1 rate=83.6788
log2c=3 log2g=-11 rate=74.5969
log2c=5 log2g=-5 rate=78.1264
log2c=-1 log2g=-5 rate=74.6343
log2c=11 log2g=-5 rate=79.7759
log2c=-3 log2g=-5 rate=74.5969
log2c=9 log2g=-5 rate=79.1846
log2c=3 log2g=-5 rate=76.8192
log2c=15 log2g=-7 rate=80.6349
log2c=15 log2g=-1 rate=85.098
log2c=15 log2g=-13 rate=77.9832
log2c=15 log2g=1 rate=87.2082
log2c=15 log2g=-11 rate=78.5372
log2c=15 log2g=-5 rate=82.3654
log2c=5 log2g=-15 rate=74.5969
log2c=-1 log2g=-15 rate=74.5969
log2c=11 log2g=-15 rate=73.9434
log2c=-3 log2g=-15 rate=74.5969
log2c=9 log2g=-15 rate=74.6281
log2c=3 log2g=-15 rate=74.5969
log2c=15 log2g=-15 rate=78.145
log2c=-5 log2g=-7 rate=74.5969
log2c=-5 log2g=-1 rate=74.5969
log2c=-5 log2g=-13 rate=74.5969
log2c=-5 log2g=1 rate=77.423
log2c=-5 log2g=-11 rate=74.5969
log2c=-5 log2g=-5 rate=74.5969
log2c=-5 log2g=-15 rate=74.5969
log2c=5 log2g=3 rate=87.1522
log2c=-1 log2g=3 rate=84.3075
log2c=11 log2g=3 rate=87.3265
log2c=-3 log2g=3 rate=82.2098
log2c=9 log2g=3 rate=87.2518
log2c=3 log2g=3 rate=86.4364
log2c=15 log2g=3 rate=87.4883
log2c=-5 log2g=3 rate=79.5456
log2c=7 log2g=-7 rate=77.6782
log2c=7 log2g=-1 rate=83.0252
log2c=7 log2g=-13 rate=74.6281
log2c=7 log2g=1 rate=85.0047
log2c=7 log2g=-11 rate=74.2484
log2c=7 log2g=-5 rate=78.8235
log2c=7 log2g=-15 rate=74.5969
log2c=7 log2g=3 rate=87.258
log2c=5 log2g=-9 rate=74.2546
log2c=-1 log2g=-9 rate=74.5969
log2c=11 log2g=-9 rate=77.9023
log2c=-3 log2g=-9 rate=74.5969
log2c=9 log2g=-9 rate=77.4479
log2c=3 log2g=-9 rate=74.6218
log2c=15 log2g=-9 rate=79.1285
log2c=-5 log2g=-9 rate=74.5969
log2c=7 log2g=-9 rate=76.5142
log2c=1 log2g=-7 rate=74.6281
log2c=1 log2g=-1 rate=79.3277
log2c=1 log2g=-13 rate=74.5969
log2c=1 log2g=1 rate=82.8447
log2c=1 log2g=-11 rate=74.5969
log2c=1 log2g=-5 rate=74.1488
log2c=1 log2g=-15 rate=74.5969
log2c=1 log2g=3 rate=85.4653
log2c=1 log2g=-9 rate=74.5969
log2c=5 log2g=-3 rate=79.2219
log2c=-1 log2g=-3 rate=74.5036
log2c=11 log2g=-3 rate=82.3965
log2c=-3 log2g=-3 rate=74.6716
log2c=9 log2g=-3 rate=80.803
log2c=3 log2g=-3 rate=78.7426
log2c=15 log2g=-3 rate=84.0959
log2c=-5 log2g=-3 rate=74.5969
log2c=7 log2g=-3 rate=79.9378
log2c=1 log2g=-3 rate=77.5537
log2c=13 log2g=-7 rate=79.2468
log2c=13 log2g=-1 rate=84.3448
log2c=13 log2g=-13 rate=77.7155
log2c=13 log2g=1 rate=87.3389
log2c=13 log2g=-11 rate=78.2633
log2c=13 log2g=-5 rate=80.6162
log2c=13 log2g=-15 rate=76.8067
log2c=13 log2g=3 rate=87.5257
log2c=13 log2g=-9 rate=78.3505
log2c=13 log2g=-3 rate=83.0252
